### PR TITLE
Tab: fix `scrollToActiveTab` method broken for vertical tabs

### LINKED ISSUE #14292: [Bug Report] ElTabs TabsNav `scrollToActiveTab` method broken for vertical tabs
### Element UI version
2.5.4

### OS/Browsers version
All

### Vue version
2.5.22

### Reproduction Link
[https://jsfiddle.net/otzLqu50/](https://jsfiddle.net/otzLqu50/)

### Steps to reproduce
1. Create an ElTabs instance with the prop `tab-position` set to `left` or `right`.
2. Set the height of the tabs so that scrolling becomes enabled.
3. Click on the down arrow to scroll through the vertical tabs.
4. Click on one of the tabs.

### What is Expected?
Tab navigation should remain in place and open the click tab's contents.

### What is actually happening?
The clicked tab's contents are opened, but the tab navigation incorrectly calculates the offset for the `scrollToActiveTab` method and scrolls to the top.

### PR DESCRIPTION
**Pull request by sinchang**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

Fixes #14292

### CODE REVIEW

File: packages/tabs/src/tab-nav.vue [hunk 1]
```diff
@@ -43,16 +43,25 @@
...
       },
       sizeName() {
-        return ['top', 'bottom'].indexOf(this.rootTabs.tabPosition) !== -1 ? 'width' : 'height';
+        return this.isHorizontal ? 'width' : 'height';
+      },
+      isHorizontal() {
+        return ['top', 'bottom'].indexOf(this.rootTabs.tabPosition) !== -1;
+      }
+    },
+    watch: {
```
Comment (iamkun) [inline packages/tabs/src/tab-nav.vue]: seems we don't need a watcher here

Comment (sinchang) [reply packages/tabs/src/tab-nav.vue]: without these codes can't working if you switch `bottom -> top -> right` in this [case](https://jsfiddle.net/otzLqu50/)

Comment (iamkun) [reply packages/tabs/src/tab-nav.vue]: I tried `bottom -> top -> right` works fine without this watcher. Any reproduction link, please?

Comment (sinchang) [reply packages/tabs/src/tab-nav.vue]: should click `tab11` first.  reprod link: https://jsfiddle.net/xu3jqc6w/

**Comment by luckyCao** [comment]
inactive

**Pull request closed without merging**